Application/Bin/VQAInteractionScreen.py

```diff
@@ -25,7 +28,15 @@ def __init__(self, threadManager, controller, models, parent=None):
         self.controller = controller
         self.models = models
         self.currentImage = None
+        self.currentQuestion = ""
+        self.currentModelImage = None
         self.predictionResult = None
+        self.exported_dir = "exported_results/"
```

Comment:
As mentioned above, specify that it's both exports and snapshots. Also capitalize & add spacing

eg. "Snapshots & Exports", "Exports & Snapshots", "Exports and Snapshots", etc.